spec/setup.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable no-console */
+const fs = require('fs');
```

Comment:
Let us add instructions on how to run this as well as a copyright license here and everywhere else.

Also, can you provide a set of points on what you have tried and what are the issues you are facing with newrelic while running the tests (You can create a doc as well).

Reply:
This can be run by running `npm run test`.
I added it to the test script in package.json https://github.com/oppia/oppiabot/blob/6c02cbd51d7bec70a8ced69fa26770c69ebaae16/package.json#L12

Reply:
Hmm, so the test is written in a way that considers `.env` is already present in the oppiabot folder. Let us update it so that it creates `.env` using `.env.example` if it is missing.

Reply:
Done.